pydantic/typing.py

```diff
@@ -271,7 +271,7 @@ def is_union(tp: Optional[Type[Any]]) -> bool:
     MappingIntStrAny = Mapping[IntStr, Any]
     CallableGenerator = Generator[AnyCallable, None, None]
     ReprArgs = Sequence[Tuple[Optional[str], Any]]
-    AnyClassMethod = classmethod[Any]
+    AnyClassMethod = classmethod[Any, Any, Any]
```

Comment:
Let's try to get CI running

```suggestion
    # classmethod[TargetType, CallableParamSpecType, CallableReturnType]
    AnyClassMethod = classmethod[Any, Any, Any]
```